bin/email_alert_service

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env ruby
+require 'govuk_app_config/configure'
```

Comment:
I suspect this needs to be `require 'govuk_app_config'` to require all the dependencies.

Reply:
Done